Halimah_Alshahrani/week_01/day_04/Homework_04/main.js

```diff
@@ -0,0 +1,101 @@
+
+//Part 1
+
+//1
+
+function sumArray(numbers) {
+	var totalSum = 0;
+	for (var i = 0; i < numbers.length; i++) {
+		totalSum += numbers[i];
+	}
+	return totalSum;
+}
+
+console.log(sumArray([2,4,6,8]));
+
+
+function multiplyArray(numbers) {
+	var totalMultiply = 1;
+	for (var i = 0; i < numbers.length; i++) {
+		totalMultiply = (totalMultiply* numbers[i]);
+	}
+	return totalMultiply;
+}
+
+console.log(multiplyArray([2,4,6,8]));
+
+//2
+ 
+function reverse(string) {
+    var text = '';
+    
+    for (var i = string.length - 1; i >= 0; i--) {
+    	text += string[i];}
+    return text;
+}
+console.log(reverse("!aloh"));
+
+//3
+
+var words = ["Hola!", "Hello", "Hi", "Hello world" ];
+
+
+function findLongestWord(string){
```

Comment:
Your logic is almost correct here, but I'd review it again and check the function definition.